packages/help-extension/src/index.tsx

```diff
@@ -135,7 +135,11 @@ const about: JupyterFrontEndPlugin<void> = {
         );
         const copyright = (
           <span className="jp-About-copyright">
-            {trans.__('© 2015-2023 Project Jupyter Contributors')}
+            {trans.__(
+              '© %1-%2 Project Jupyter Contributors',
+              2015,
+              new Date().getUTCFullYear()
```

Comment:
What if user's browser has a mis-configured calendar? This could even end up being `© 2015-1982` or so. I think dynamic year is fine as long as this is generated on release time and fed down here. That said, even if we keep it as hard-coded in this PR, the change to use substitution in the translation string is a welcome one as it reduces the burden on translators

Reply:
Thanks for sharing your perspective, @krassowski! Is the following snippet your suggestion?

```tsx
{trans.__(
  '© %1-%2 Project Jupyter Contributors',
  2015,
  2024
)}
```

If yes, let me know if I should update the PR with this change, please.

Reply:
I think for now that is a safe bet, yes